core/src/main/resources/hudson/model/Computer/setOfflineCause.jelly

```diff
@@ -26,6 +26,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <l:layout title="${%title(it.displayName)}">
     <st:include page="sidepanel.jelly" />
+    <l:breadcrumb title="${%title(it.displayName)}" />
```

Comment:
unless there's a better text possibly whatever the link text is that gets you to the page:
```suggestion
    <l:breadcrumb title="${%Update offline reason}" />
```